cogs5e/models/sheet/base.py

```diff
@@ -28,27 +28,40 @@ def to_dict(self):
     @classmethod
     def default(cls):
         return cls(0, 10, 10, 10, 10, 10, 10)
+    
+    @classmethod
+    def custstats(cls,cust,origstats):
+        stats=[]
+        abils=('pro',)+STAT_ABBREVIATIONS
+        for i in range(len(abils)):
+            stat=abils[i]
+            stats.append(cust[stat] if stat in cust.keys() else origstats.get_mod(stat))
+        return cls(stats[0],stats[1],stats[2],stats[3],stats[4],stats[5],stats[6])
+            
 
     def get_mod(self, stat: str):
         abbr_stat = stat.lower()[:3]
-        if abbr_stat not in STAT_ABBREVIATIONS:
+        abils=('pro',)+STAT_ABBREVIATIONS
+        if abbr_stat not in abils:
             raise ValueError(f"{stat} is not a valid stat.")
         return {
-            'str': self.strength // 2 - 5, 'dex': self.dexterity // 2 - 5,
+            'pro': self.prof_bonus, 'str': self.strength // 2 - 5, 'dex': self.dexterity // 2 - 5,
```

Comment:
This is a potentially breaking change that appears to be used for the `.custstats()` classmethod. See comment on that method's definition.